transport/src/main/java/io/netty/channel/pool/FixedChannelPool.java

```diff
@@ -327,7 +336,7 @@ private void decrementAndRunTaskQueue() {
         runTaskQueue();
     }
 
-    private void runTaskQueue() {
+    private void runTaskQueue() throws Exception {
```

Comment:
Doesn't look like this change is needed and subsequently the change above to `decrementAndRunTaskQueue()`

Reply:
Done